libs/core/async_mpi/tests/unit/algorithm_transform_mpi.cpp

```diff
@@ -212,8 +212,11 @@ int hpx_main()
                 }
                 catch (std::runtime_error const& e)
                 {
-                    HPX_TEST(std::string(e.what()).find(std::string(
-                                 "invalid root")) != std::string::npos);
+                    auto error_msg = std::string(e.what());
+                    for (auto& c : error_msg)
```

Comment:
We do have this: https://github.com/STEllAR-GROUP/hpx/blob/master/libs/core/string_util/include/hpx/string_util/case_conv.hpp.